docker/Dockerfile

```diff
@@ -45,8 +45,11 @@ FROM python:${PYTHON_VERSION}-slim-bullseye AS runner-image
 RUN apt-get update && \
     apt-get install --no-install-recommends -y \
         curl \
+        ffmpeg \
         gifsicle \
-        libcairo2 && \
+        libcairo2 \
+        libexiv2-27 \
```

Comment:
Thanks for the PR, @Tenzer.
To keep the image small, I would remove FFmpeg, since it's only needed when converting Gif to mp4/gifv.
Adding `libexiv2-27` doesn't make `pyexiv2` work, so I would also remove it.

Reply:
Cool. I have removed ffmpeg and libexiv2 which means this only ends up adding jpegtran to the runtime image, which I guess is still an improvement.